app/src/test/java/lang_import/org/app/FeedReaderTest.kt

```diff
@@ -0,0 +1,19 @@
+package lang_import.org.app
+
+import android.content.Context
+import org.junit.Test
+
+import org.junit.Assert.*
+import java.io.File
+
+val sampleText = File("src/test/resources/habr.rss.xml").readText()
```

Comment:
Not worked offline (or if server is not response), mb will need create local test.xml :)